src/inputs/datePickerInput.jsx

```diff
@@ -37,11 +91,14 @@ class DatePickerInput extends React.PureComponent {
         let inputValue;
 
         if (props.rangeFrom) {
-            inputValue = this._safeDateFormat(newDateFrom, props.locale);
+            // eslint-disable-next-line no-underscore-dangle
```

Comment:
Looks like we solved the issue.  No need for suppression comments anymore, right?

Reply:
Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.